core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1147,7 +1142,7 @@ create_sip_endpoint(Endpoint, Properties, #clid{}=Clid, Call) ->
                       ,{<<"Endpoint-Actions">>, endpoint_actions(Endpoint, Call)}
                        | maybe_get_t38(Endpoint, Call)
                       ])),
-    maybe_format_endpoint(SIPEndpoint, kz_term:is_empty(kz_json:get_json_value(<<"formatters">>, Endpoint))).
+    maybe_format_endpoint(SIPEndpoint, kz_json:get_json_value(<<"formatters">>, Endpoint)).
```

Comment:
can you use `kz_json:get_ne_json_value/2` here to make sure it's not empty?

Reply:
The iteration in kz_formatters just folds over the empty Formatters as a no-op.